hrm/department_app/urls.py

```diff
@@ -0,0 +1,29 @@
+"""URLS"""
+from django.urls import path, include
+from department_app.views.department_views import DepartmentViewSet
+from department_app.views.employee_views import EmployeetViewSet
+from department_app.views.user_views import LoginAPI, RegisterAPI, UserAPI, ChangePasswordView
+from knox import views as knox_views
+from rest_framework.routers import DefaultRouter
+
+
+app_name = 'department_app'
+
+router = DefaultRouter()
+router.register('department', DepartmentViewSet, basename='departments')
+router.register('employee', EmployeetViewSet, basename='employees')
+
+urlpatterns = [
+    path('viewset/', include(router.urls)),
+    path('viewset/<int:pk>/', include(router.urls)),
```

Comment:
What is the purpose of this endpoint? Django and ViewSets automatically generate URLs for manipulating records by primary key.

Reply:
I thought it would not be superfluous to specify it explicitly.I deleted it.